tslint.json

```diff
@@ -8,7 +8,8 @@
     "rules": {
         "comment-format": false,
         "member-ordering": false,
-
+        // Allow throws w/o stack trace
+        "no-string-throw": false,
```

Comment:
Why did we need to enable this?  Are we throwing strings or some dependency?

If it's our code I think we should throw `Error` objects instead of strings